src/poggit/ci/builder/DefaultProjectBuilder.php

```diff
@@ -197,11 +198,15 @@ protected function build(Phar $phar, RepoZipball $zipball, WebhookProjectModel $
         }elseif(sizeof($apis) === 1){
             $majorApis[] = $apis[0][0];
         }else{
+            $redundant = false;
             foreach($apis as $api){
                 if(!in_array($api[0], $majorApis)){
                     $majorApis[] = $api[0];
-                }else{
-                    //TODO: REDUNDANT API LINT
+                }elseif($redundant === false and stripos($api, "-beta", 4) === false and stripos($api, "-alpha", 4) === false){
```

Comment:
what does the 4 signify?

also, the suffixes are no longer checked since PM4

Reply:
Just a starting index, so skip major.minor.patch

Not really needed tbh not a massive string lmao.

Oh brilliant, I'll scrap it.

Reply:
> Just a starting index, so skip major.minor.patch
>Not really needed tbh not a massive string lmao.

not really needed and premature optimisation, it might break if a version like `4.0-beta` was used

Reply:
`4.0-beta` isn't even semver right?